uint8 servo_id
uint8 command_type  # CMD_WRITE_B3M / CMD_SET_POS_B3M
B3mSetPosCmd cmd_set_pos
B3mWriteCmd cmd_write

uint8 CMD_WRITE = 0
uint8 CMD_SET_POS = 1
